static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -103,6 +104,7 @@ if (!commercialFeatures.adFree) {
 		['cm-redplanet', initRedplanet],
 		['cm-paidContainers', paidContainers],
 		['cm-commentAdverts', initCommentAdverts],
```

Comment:
Do we still need this `initCommentAdverts` module - it's possibly broken anyway if it relies on Mediator.

Reply:
I did have a look at this and got down a rabbit hole of deleting a large amount of code. Then realised that not having an ideal testing situation right now makes this probably not the best time to remove it. I shall create a ticket. There is a comment to add `amIUsed` so maybe this is a first step.

Reply:
I did some investigation around that a while back, can't find the card! But it may still be used on those few and far between frontend rendered pages

Reply:
Sure, that makes sense! `amIUsed` sounds like a good idea.